webcompat/api/endpoints.py

```diff
@@ -170,22 +169,32 @@ def get_search_results(query_string=None, params=None):
 
 
 @api_bp.route('/issues/<int:number>/comments', methods=['GET', 'POST'])
-@mockable_response
 def proxy_comments(number):
-    """XHR endpoint to get issues comments from GitHub.
+    """XHR endpoint to get issues comments from GitHub, or to add
+    a new comment.
 
     Either as an authed user, or as one of our proxy bots.
     """
```

Comment:
nit: first docstring line should be only one line.

```suggestion
    """XHR endpoint for GitHub issue comments.
    
    * GET an issue comments 
    * POST a comment on an issue (only as an authorized GitHub user)
    """
```